packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -16,6 +16,33 @@ import patch from '../patch'
 import readChunk from '../_readStreamChunk'
 import { generateToken } from '../utils'
 
+// networks field contains:
+// - <n>/ip <=> <n>/ipv4/0
+// - <n>/ipv4/<m>
+// - <n>/ipv6/<m>
+// where n is the VIF order and m is the IP order
```

Comment:
I don't think this comment is really necessary, it's understandable from the linked commetn and from this code.

Reply:
IMHO, it isn't obvious.